.gitignore

```diff
@@ -16,6 +24,14 @@ HasalServer.log.*
 stage/*
 docs/*
 
+# File types to ignore
+*.zip
+*.diff
+*.json
```

Comment:
`settings` folder also has some json config files.
AFAIK, @ShakoHo will add more json config files to control the parameters in the future.

@ShakoHo , do you have any suggestions?